.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 
 rvm:
-  - 2.2.3
+  - 2.2.5
```

Comment:
Due to having `guard` as a development dependency. It's pulling in ruby_dep (https://rubygems.org/gems/ruby_dep/versions/1.4.0). Which now requires ruby version >= 2.2.5.

Bumped the travis version so that it wouldn't break the build.